bin/ncp/BACKUPS/nc-backup-auto.sh

```diff
@@ -7,6 +7,17 @@
 # More at https://ownyourbits.com/2017/02/13/nextcloud-ready-raspberry-pi-image/
 #
 
+tmpl_get_destination() {
+  (
+  . /usr/local/etc/library.sh
+  find_app_param nc-backup-auto DESTDIR
+  )
+}
+
+is_active() {
```

Comment:
this should not be necessary

Reply:
Alright, I decided against using it to toggle metrics anyways (there's no reason to stop reporting backup freshness, if backups aren't scheduled - quite few opposite, actually). I'll remove it tomorrow. 👍